docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,267 @@
+[[semantic-search-semantic-text]]
+=== Tutorial: semantic search with `semantic_text`
+++++
+<titleabbrev>Semantic search with `semantic_text`</titleabbrev>
+++++
+
+beta[]
+
+This tutorial shows you how to use the semantic text feature to perform semantic search on your data.
```

Comment:
Can we create a notebook that goes through these steps and link to it here?

Reply:
Yes, creating a notebook is on my TO-DO list. I'll do it in a separate PR and publish it after the tutorial docs. Will update the documentation with the link after that.